bloom-housing/bloom
# Update active/closed listing visibility

# Pull Request Template

## Issue

Addresses #1576 (issue)


## Description

It changes the listing component to filter active/closed listing by `status` property, instead of `dueDate` time.

## Type of change


## How Can This Be Tested/Reviewed?

1. Open listings list in the public app
2. Open partners and close a listing
3. The listing should be visible in the "CLOSED LISTINGS" section
3. Publish the listing
4. The listing should be visible as active, should not be in the "CLOSED LISTINGS" section


## Checklist:
